web/packages/teleterm/src/ui/services/clusters/clustersService.ts

```diff
@@ -45,6 +45,7 @@ import * as uri from 'teleterm/ui/uri';
 import { NotificationsService } from 'teleterm/ui/services/notifications';
 import { MainProcessClient } from 'teleterm/mainProcess/types';
 import { UsageService } from 'teleterm/ui/services/usage';
+import { AssumedRequest } from 'teleterm/services/tshd/types';
```

Comment:
It says that this type is deprecated and to use `AccessRequest` instead, directly imported from `gen-proto-ts`.

Reply:
I can't change it here unfortunately, this method returns `cluster.loggedInUser.assumedRequests` which has a type of `AssumedRequest`. This part of code has to be removed, in general, as per a TODO:
```
// TODO(gzdunek): Remove assumedRequests from loggedInUser.
// The AssumedRequest objects are needed only in AssumedRolesBar.
// We should be able to move fetching them there.
```